ui-components/src/page_components/listing/ListingsList.tsx

```diff
@@ -10,10 +10,12 @@ import "./ListingsList.scss"
 
 export interface ListingsProps {
   listings: Listing[]
+  hideApplicationStatus?: boolean
 }
 
 const ListingsList = (props: ListingsProps) => {
   const listings = props.listings
+  const hideApplicationStatus = props.hideApplicationStatus || false
```

Comment:
I don't think you need this, since the prop will just evaluate to false if it's not defined.